pandas/tests/indexes/test_base.py

```diff
@@ -996,30 +958,38 @@ def test_iadd_string(self):
         index += '_x'
         assert 'a_x' in index
 
-    def test_difference(self):
-
+    @pytest.mark.parametrize("second_nm,exp", [
+        (None, None), ('name', 'name')])
```

Comment:
Might be over-engineering the parameters in this case, but figure if we wanted to expand to cover more scenarios this is a better framework to do so

Reply:
ok by me